se2_planning/src/GridMapStateValidator.cpp

```diff
@@ -30,6 +30,15 @@ void GridMapStateValidator::setGridMap(const grid_map::GridMap& gridMap) {
     throw std::runtime_error("Grid map has no layers");
   }
   gridMap_ = gridMap;
+  // TODO does not work correctly, boundaries of traversability layer get set to nan!? Problem because elevation_mapping_cupy
+  //  has strange resolution?!
+  // Convert resolution, optionally only copy relevant layer
+  //  gridMap_.setFrameId(gridMap.getFrameId());
```

Comment:
is this still an issue? if this is an issue of cupy then we should maybe address it somewhere else

Reply:
As far as I remember, the issue was in the grid map itself. I have to check again. A function somehow does not work as expected (maybe the addDataFrom()). I'll look into that.

Reply:
The commented code should not be needed. I removed it.